src/hooks/api/useApiVaultDynamic.ts

```diff
@@ -2,14 +2,13 @@ import { useQuery } from "@tanstack/react-query"
 import { Address, useAccount } from "wagmi"
```

Comment:
Reasons for renaming: its easier to type `useApiVault...` or `useVault...` than `useApiCompounder...` and `useCompounderPool...`